src/components/Input/index.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { FunctionComponent } from 'react';
+
+// Custom hooks
+import { useAppSelector } from '../../redux/hooks';
+
+// Styles
+import * as S from './styles';
+
+const INPUT = {
+  input: ({ type, placeholder, theme }: { type: string; placeholder: string; theme: string }) => (
+    <S.Input type={type} placeholder={placeholder} theme={theme} />
+  )
+};
+
+interface InputProps {
+  inputElement: 'input';
```

Comment:
are you going to extend available set of values for 'inputElement' prop?

Reply:
Yes. Since there will be a feedback page with a form which will possibly include textarea elements. I haven't decided on this yet.